### PR TITLE
Get the correct ProgFolder for Arm64

### LINKED ISSUE #409: Fix  pal::get_default_servicing_directory
## Steps to reproduce

run a standalone and a portable app cross-published from the CLI using a Win10-x64 box for a win10-arm64 RID.
 
Both apps run and iget the message I was expecting, however, I also got some extra:
 

## Expected  behavior
Hello World!

## Actual behavior
Error resolving full path []
Error resolving full path []
Hello World!

### PR DESCRIPTION
**Pull request by ramarag**
Fixes #409 
@gkhanna79  @schellap PTAL.

### CODE REVIEW

File: src/corehost/common/pal.windows.cpp [hunk 1]
```diff
@@ -157,11 +157,12 @@ bool pal::get_default_servicing_directory(string_t* recv)
     recv->clear();
 
     // See https://github.com/dotnet/cli/issues/2179
-#ifdef _TARGET_X86_
+#if defined(_TARGET_AMD64_)
+    if (!pal::getenv(_X("ProgramFiles(x86)"), recv))
+#else
     // In WOW64 mode, PF maps to PFx86.
+    // For Arm, there is no wow mode as of now, so this should be fine
```
Comment (gkhanna79) [inline src/corehost/common/pal.windows.cpp]: Typo: Arm64.

Comment (ramarag) [inline src/corehost/common/pal.windows.cpp]: fixed it

**Comment by gkhanna79** [comment]
can you please request Arm64 CI build as well?

**Comment by ramarag** [comment]
@dotnet-bot test windows_NT ARM64 Please

**Comment by ramarag** [comment]
@dotnet-bot test windows_NT release arm64 please

**Comment by ramarag** [comment]
@dotnet-bot test Windows_NT Release arm64 please

**Comment by ianhays** [comment]
@dotnet-bot test ARM64 Windows_NT Release

**Comment by ramarag** [comment]
@gkhanna79 the Arm64 build has completed successfully

**Pull request merged**